pulsebot/bugzilla.py

```diff
@@ -23,47 +23,52 @@ def __init__(self, server, api_key):
             "User-Agent": "pulsebot",
             "X-Bugzilla-API-Key": api_key,
             "Content-Type": "application/json",
-            "Accept": "application/json"
+            "Accept": "application/json",
         }
 
-    def _call(self, method, path, data=None):
+    def _call(self, method, path, params=None, data=None):
+        method = method.lower() or 'get'
```

Comment:
This would cause an error if no method is passed (because of `.lower`). You could do this instead
```suggestion
        method = method.lower() if method else 'get'
```